tests/features/reports.feature

```diff
@@ -1,11 +1,30 @@
 @reports
 Feature: Reports show the right things
 
-Note: For now we'll have to check them visually
-
 @fast @r1
-Scenario: Report still has page id when I tap to continue without setting any fields
-  Given I start the interview at "all_tests"
+Scenario: Report still shows page id when I tap to continue without setting any fields
+  Given
```

Comment:
I think this should say "Given the Scenario should pass with this report:"

Reply:
It definitely should! Wonder why it's passing. Will have to fix that for sure.